uint8[20] line1
uint8[20] line2
uint8[20] line3
uint8[20] line4
